simul/build.go

```diff
@@ -150,31 +146,20 @@ func RunTests(name string, runconfigs []*platform.RunConfig) {
 
 		// run test t nTimes times
 		// take the average of all successful runs
-		runs := make([]*monitor.Stats, 0, nTimes)
-		for r := 0; r < nTimes; r++ {
-			stats, err := RunTest(rc)
-			if err != nil {
-				log.Error("Error running test, trying again:", err)
-				continue
-			}
-
-			runs = append(runs, stats)
-			if stopOnSuccess {
-				break
-			}
-		}
-
-		if len(runs) == 0 {
-			log.Lvl1("unable to get any data for test:", rc)
+		stats, err := RunTest(rc)
+		if err != nil {
+			log.Error("Error running test, trying again:", err)
 			continue
 		}
 
-		s := monitor.AverageStats(runs)
 		if i == 0 {
-			s.WriteHeader(f)
+			stats.WriteHeader(f)
+		}
+		if strings.ToLower(rc.Get("EveryRoundStats")) == "true" {
```

Comment:
Can we call it more like `IndividualMeasures` (to be an antonym of `aggregate measures` ? 
And just `if rc.Get("IndividualMeasures") != "" { ... }` should be enough ?